sdk-python/test/openai_filter_test.py

```diff
@@ -0,0 +1,62 @@
+import io
+import logging
+import unittest
+from omnilog.start_listener import OpenAIFilter
+
+
+class TestOpenAIFilter(unittest.TestCase):
+    input: str
+    result: str
+
+    def compute_result(self, name):
+        parent = logging.getLogger(name)
```

Comment:
Is Omnilog being mocked by the parent or child in this case? Or am I misunderstanding?

Reply:
The child is the one to receive the input, so the child is sys.stdout, and thus the parent is OmniLog. The logs propagate upwards